### PR TITLE
Re add imu lidar

### PR DESCRIPTION
**Pull request by jalesseVolpe**
This is the re-add of the lidar and IMU. Please review and let me know if anything is wrong. Thanks

### CODE REVIEW

File: c1tenth_drivers/launch/c1tenth_drivers.launch.py [hunk 1]
```diff
@@ -33,6 +33,91 @@
...
+        parameters=[SLLIDAR_DRIVER_PARAM_FILE],
+        output='screen',
+        remappings=[
+            ('scan', 'scan_raw')
+        ]
+    )
+
+
+  # Subscribes to:
+  #   points_raw
```
Comment (asymingt) [inline c1tenth_drivers/launch/c1tenth_drivers.launch.py]: nit: please add ROS message type

**Review by asymingt — APPROVED** [review]

File: c1tenth_drivers/launch/c1tenth_drivers.launch.py (hunk 1 above)
Comment (jalesseVolpe) [reply c1tenth_drivers/launch/c1tenth_drivers.launch.py]: sure.. will do within next hour

Comment (jalesseVolpe) [reply c1tenth_drivers/launch/c1tenth_drivers.launch.py]: Added and pushed

**Comment by jalesseVolpe** [comment]
thanks

**Pull request merged**